game.py

```diff
@@ -20,239 +20,273 @@
 # Contact information:
 # Utkarsh Tiwari    iamutkarshtiwari@gmail.com
 
+import os
+from gettext import gettext as _
+
 import gi
 gi.require_version('Gtk', '3.0')
 from gi.repository import Gtk
 import pygame
-import sys
-from gettext import gettext as _
 
-from elements import *
-from welcomescreen import *
-from scorescreen import *
+from elements import Element
+from welcomescreen import Welcomescreen
+from scorescreen import Scorescreen
+
+FPS = 60
+
+# Game states
+WELCOME = 0
+RULES = 1
+PLAY = 2
+PAUSE = 3
+
+BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+
+# SPEED of cars in the x-axis, lane change speed
+ANGLE_LIMIT = 30
+ANGLE_SPEED = 2
+
+TICK_COUNTER_THRESHOLD = 40
+
+FONT_PATH_ARIMO = os.path.join("fonts", "Arimo.ttf")
+
+BACKGROUND_IMAGE_PATH = os.path.join("assets", "background.png")
+REDCAR_IMAGE_PATH = os.path.join("assets", "redcar.png")
+BLUECAR_IMAGE_PATH = os.path.join("assets", "bluecar.png")
+
+SOUND_HIT_PATH = os.path.join("assets", "sounds", "hit.ogg")
+SOUND_MISS_PATH = os.path.join("assets", "sounds", "miss.ogg")
+SOUND_MUSIC_PATH = os.path.join("assets", "sounds", "music.ogg")
+SOUND_SCORE_PATH = os.path.join("assets", "sounds", "score.ogg")
+
+RED_CIRCLE_PATH = os.path.join("assets", "redcircle.png")
+BLUE_CIRCLE_PATH = os.path.join("assets", "bluecircle.png")
+RED_SQUARE_PATH = os.path.join("assets", "redsquare.png")
+BLUE_SQUARE_PATH = os.path.join("assets", "bluesquare.png")
 
 
 class Game:
+<<<<<<< HEAD:game.py
 
+=======
+>>>>>>> 34c7954 (Refactor activity):main.py
```

Comment:
Forget to remove these?

Reply:
Leftover from the rebase, I thought I couldn't continue rebase untill all of those were resolve, aparently i was wrong. Anyways fixed